Learn-CSS-Blog/index5.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+</head>
+<body>
+</body>
+</html>
+
+<!-- 
+    Create a `div` element. Make sure the opening `<div>` and closing `</div>`
```

Comment:
Create a `<div>` element. Make sure the opening `<div>` and closing `</div>`
tags are between the `<body>` tags.